assignments/array-methods.js

```diff
@@ -55,29 +55,66 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
-let fullName = [];
+// let fullName = []
+// for (i = 0; i < runners.length; i++) {
+//     n = runners[i].first_name +' '+runners[i].last_name;
+//     fullName.push(n);
+// }
+fullName = [];
+runners.forEach( function(currentValue){
+    fullName.push(currentValue.first_name+' '+currentValue.last_name);
+})
 console.log(fullName);
-
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
-let allCaps = [];
+// let allCaps = [];
+const allCaps = runners.map( function(currentValue){
+    return currentValue.first_name.toUpperCase();
+});
 console.log(allCaps); 
 
 // ==== Challenge 3: Use .filter() ====
 // The large shirts won't be available for the event due to an ordering issue.  Get a list of runners with large sized shirts so they can choose a different size. Return an array named largeShirts that contains information about the runners that have a shirt size of L and log the result
-let largeShirts = [];
+// largeShirts = []
+// for (i = 0; i < runners.length; i++){
+    
+//     if (runners[i].shirt_size === 'L') {
+//         n = runners[i].first_name;
+//         largeShirts.push(n);
+        
+//     }
+// }
+const largeShirts = runners.filter( function(currentValue){
+    return currentValue.shirt_size == 'L';
```

Comment:
lookin good but I would try to stick to a triple = which is more precise, just incase using the double bites you in the butt later down the road (though there are times you only care about lose equality)